examples/annotate-config/.krmignore

```diff
@@ -0,0 +1 @@
+.expected
```

Comment:
What if we don't have this line here?
What if there are no yaml files in .expected? When do we have to have this line?

Reply:
kpt will read the yaml files from .expected (results.yaml and config.yaml) and process them just like they are regular k8s resources. It's better to always have this line but it can be ignored when there is no yaml file in .expected.